src/ansiblelint/__main__.py

```diff
@@ -452,6 +452,7 @@ def path_inject(own_location: str = "") -> None:
         inject_paths.append(str(py_path))
 
     # last option, if nothing else is found, just look next to ourselves...
+    own_location = os.path.realpath(own_location)
```

Comment:
This breaks #3903 again, do we have to resolve symlinks here? Maybe the resolved location can only be used where needed?